### PR TITLE
Milestone 4 : Add Styles

### PR DESCRIPTION
**Pull request by eroiyo**
Hello Microverse Community, this is the milestone where we add the styles inside the page, all the css rules were added, and some elements for decorations too.

*Changed the genre input for a selector
*Styles have been added to all the page

It says "Coming soon" in the author section because the API dont have the capacity to save author's name, so I will add it later when I am able to use other API.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -2,20 +2,22 @@ import React from 'react';
...
+  <header>
+    <ul className="navbar">
+      <li>
+        <Link to="/" className="Logo booksButton">Bookstore CMS</Link>
+      </li>
+      <li>
+        <Link to="/" className="navLink booksButton">BOOKS</Link>
+      </li>
+      <li>
+        <Link to="/categories" className="navLink categoriesButton">CATEGORIES</Link>
```
Comment (ZawadiSumba66) [inline src/components/Header.js:14]: - [ ] The categories link is a little bit greyish as seen from the design. Kindly fix this.

File: src/components/Header.js [hunk 2]
```diff
@@ -2,20 +2,22 @@ import React from 'react';
...
+      </li>
+    </ul>
+    <button className="portrait" type="button">
+      <i className="fa fa-user user" aria-hidden="true" />
+    </button>
```
Comment (ZawadiSumba66) [inline src/components/Header.js:19]: - [ ] The potrait link is not showing as expected. It should be flexed to the right, be a user icon and blue in color. Lets work on this. 💪

File: src/components/Books/Form.js [hunk 3]
```diff
@@ -26,10 +26,19 @@ const Form = () => {
...
   return (
     <form onSubmit={submitBook}>
-      <input placeholder="title" onChange={changeTitle} type="text" id="book-title" />
-      <input placeholder="genre" onChange={changeGenre} type="text" id="book-genre" />
-      <input placeholder="author" onChange={changeAuthor} type="text" id="book-author" />
-      <button type="submit">submit</button>
+      <div className="xdivisor" />
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <input placeholder="Book Title" className="input" onChange={changeTitle} type="text" id="book-title" />
+      <select className="input genre-categorys" onChange={changeGenre} id="book-genre">
```
Comment (ZawadiSumba66) [inline src/components/Books/Form.js:32]: - [ ] The placeholders for the book input and category select input ought to be greyish in color such as the design below:
image
- [ OPTIONAL] You can clear the inputs once a user has clicked on the add book button so as to have a better user experience on your app

File: src/components/Books/Form.js [hunk 4]
```diff
@@ -26,10 +26,19 @@ const Form = () => {
   };
   return (
     <form onSubmit={submitBook}>
-      <input placeholder="title" onChange={changeTitle} type="text" id="book-title" />
-      <input placeholder="genre" onChange={changeGenre} type="text" id="book-genre" />
-      <input placeholder="author" onChange={changeAuthor} type="text" id="book-author" />
-      <button type="submit">submit</button>
+      <div className="xdivisor" />
```
Comment (ZawadiSumba66) [inline src/components/Books/Form.js:29]: - [ ] Kindly adjust this divider to be thinner as seen from the design given.

File: src/index.css [hunk 5]
```diff
@@ -1,3 +1,311 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap");
+
 body {
+  background-color: #f5f6fa;
```
Comment (ZawadiSumba66) [inline src/index.css:4]: - [ ] There is quite a few repetitions in some of the CSS selectors. Such as:
```
 font-weight: 300;
 font-family: "Roboto Slab", serif;
 font-family: "Montserrat", sans-serif;
 color: #121212;
````
You can correct this by following the DRY rule where you can assign a similar class to your JSX code where the style applies and then use that use it as a CSS selector to contain all the styles you want that element to have.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi @eroiyo 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 5 above)
Comment (eroiyo) [reply src/index.css:4]: Thanks!, i i will be working on it

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @eroiyo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Thanks!

**Pull request merged**